### DISCUSSION TITLE
Errors calling reusable workflow started today

### DISCUSSION THREAD

**Discussion by manraog** [community/community#37943] [Actions]
Hi, we have a reusable workflow that we use across several repositories. This morning everything was working fine but around 9 PM UTC on 31 Oct some workflows don't start and fail with the following message: 

`Invalid workflow file: .github/workflows/push-manual_main.yml#L1 The workflow is not valid. <ORG>/SAUTO_CICD/.github/workflows/reusable_deploy.yml@main: While scanning a literal block scalar, found extra spaces in first line. Unexpected type " encountered while reading 'root'. The type 'MappingToken' was expected. Show less
`

We have rolled back our latest changes to the reusable workflow but the problem persists. Any idea what can be? Strangely, not all workflows have the same behavior but are pointing to the same reusable's branch. 

Thanks in advance.

**Comment by genisd** [comment]
Yup same problem here:

This is the yaml:

```
"jobs":
  "build-docker":
```

this is the error

![image](https://user-images.githubusercontent.com/4093500/199207228-39bf7c17-49e9-42ae-b445-7a1e5b9969af.png)


Feels like a regression upstream

**Reply by genisd** [reply]
I created a support ticket for this, felt like the best thing i could do

**Reply by manraog** [reply]
We have created a ticket too, but right now it appears to be solved. The ticket wasn't answered.

**Reply by fhp** [reply]
Can confirm. It is working for us again as well.
